posthog/api/test/test_organization.py

```diff
@@ -103,6 +103,41 @@ def test_cannot_update_organization_if_not_owner_or_admin(self):
         self.organization.refresh_from_db()
         self.assertNotEqual(self.organization.name, "ASDFG")
 
+    def test_plugin_alerts_list(self):
+        self.organization_membership.level = OrganizationMembership.Level.OWNER
+        self.organization_membership.save()
+        user2 = self._create_user("user2@posthog.com")
+        user3 = self._create_user("user3@posthog.com")
+
+        response = self.client.patch(
+            f"/api/organizations/{self.organization.id}",
+            {"members_to_send_plugin_alerts": [self.user.uuid, user2.uuid]},
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.json()["members_to_send_plugin_alerts"], [str(self.user.uuid), str(user2.uuid)])
```

Comment:
non-blocking

If the order isn't important or isn't guaranteed, make these `set`s before comparison such that we don't risk this becoming flaky